x-pack/plugins/search_playground/server/lib/conversational_chain.test.ts

```diff
@@ -54,9 +65,11 @@ describe('conversational chain', () => {
       },
     };
 
-    const llm = new FakeListChatModel({
-      responses,
-    });
+    const llm = isChatModel
+      ? new FakeListChatModel({
+          responses,
+        })
+      : new FakeListLLM({ responses });
```

Comment:
Do you know if the lint is correct here? Looks like two different styles for two functions.